pyOCD/flash/analyzer/build.bat

```diff
@@ -0,0 +1,4 @@
+set path=%path%;C:\Program Files (x86)\GNU Tools ARM Embedded\4.9 2014q4\bin
+arm-none-eabi-gcc.exe -O3 -ffunction-sections -fdata-sections -Wall -mcpu=cortex-m0 -mthumb -mfloat-abi=soft -g3 -gdwarf-2 -gstrict-dwarf -T"linker_script.ld" -Wl,-Map,main.map,--gc-sections,-ecompute_crc -n -mcpu=cortex-m0 -mthumb -mfloat-abi=soft -g3 -nostdlib -fpic main.c -o main.elf
+arm-none-eabi-objcopy.exe --output-target binary main.elf main.bin
+python.exe generate_python.py
```

Comment:
For such a small program I think it is ok to leave it using a simple script.  Multi-platform support could be an additional improvement in the future if the algorithm needs to be rebuilt often.